### ISSUE TITLE
AiMesh

### ISSUE THREAD

**Issue by PavelBaco** [gzenux/asuswrt-rtn18u#33]
Hi, its possible to run on RT N18U AiMesh function? I install this great FW but didnt see AiMesh icon on my router.

**Comment by gzenux** [comment]
No, Asus didn't release any AiMesh related prebuilt binary (closed source) for this model.
Furthermore, it's the same reason why this model lacks some Asus features such as Adaptive QoS and AiProtection.

**Comment by levanid** [comment]
More than a year have passed, so I'll try my luck asking for Ai Mesh node functionality. 
If there's a possibility to use N18 router as Ai Mesh node (backbone connected via ethernet cable)? Maybe ASUS did update their libraries?
